index.css

```diff
@@ -11,21 +11,22 @@ body {
 .content {
   max-width: 1024px;
   margin: 0 auto;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
+ 
+}
+
+header {
+display: flex;
+flex-direction: column;
+align-items: center;
```

Comment:
Header is a block element. Its inner elements are also blocks. So, it will be column anyway in your case. + You missed a code style here again :) More accurate please.